tests/emukit/benchmarking/test_metrics.py

```diff
@@ -0,0 +1,48 @@
+import mock
+import numpy as np
+
+from emukit.benchmarking.metrics import MeanSquaredErrorMetric, MinimumObservedValueMetric
+from emukit.core.interfaces import IModel
+from emukit.core.loop import LoopState, ModelUpdater, OuterLoop
+from emukit.core.loop.loop_state import create_loop_state
+
+
+def test_mean_squared_error_metric():
```

Comment:
some test, even if it does not measure the actual output, for time metric would be nice